ansible/macos/macos/base.yml

```diff
@@ -397,6 +397,39 @@
               - run-http-proxy (compatibility)
               - run-dinghy-proxy (compatibility)
 
+    - name: Restart http-proxy service
+      tags: http-proxy
+      block:
+        - name: Check if Docker daemon is running
+          command: docker info
+          register: docker_check
+          changed_when: false
+          failed_when: false
+
+        - name: Clean up existing http-proxy container
+          command: docker rm -vf http-proxy
+          register: docker_cleanup
+          failed_when: false
+          changed_when: docker_cleanup.rc == 0 and 'http-proxy' in docker_cleanup.stdout
+          when: docker_check.rc == 0
```

Comment:
We don't need this; the spark-http-proxy command already handles that with Docker Compose.